promptsource/templates/xnli/en/templates.yaml

```diff
@@ -42,16 +51,19 @@ templates:
   d9e13133-267e-46c4-afad-c2379dcc5272: !Template
     id: d9e13133-267e-46c4-afad-c2379dcc5272
     jinja: "{{premise}}\nQuestion: {{hypothesis}} True, False, or Neither? ||| \n\
-      {% if label == 0 %} \nTrue\n{% elif label == 1 %}\nNeither\n{% else %}\nFalse\n\
-      {% endif %}"
+      {{[\"True\", \"Neither\", \"False\"][label]}}"
     name: ANLI GPT3
     reference: ANLI prompt format from Table G7 in the GPT3 paper
     task_template: true
   dd4276e6-aebd-44a3-b3cf-baf8a4c237f0: !Template
     id: dd4276e6-aebd-44a3-b3cf-baf8a4c237f0
-    jinja: "Sentence 1: {{premise}}\nSentence 2: {{hypothesis}}\nQuestion: Does Sentence\
-      \ 1 entail Sentence 2? Yes or No? |||\n{% if label == 0 %} \nYes\n{% else %}\n\
-      No\n{% endif %}"
+    jinja: 'Sentence 1: {{premise}}
+
+      Sentence 2: {{hypothesis}}
+
+      Question: Does Sentence 1 entail Sentence 2? Yes or No? |||
+
+      {{["Yes", "No", "No"][label]}}'
```

Comment:
`["Yes", "No", "No"]`, "No" twice?

Reply:
This is intentional, the original conditional mapped 0 -> yes, 1 or 2 -> no.